op-node/rollup/derive/interop_upgrade_transactions.go

```diff
@@ -0,0 +1,97 @@
+)
+
+func InteropNetworkUpgradeTransactions() ([]hexutil.Bytes, error) {
+	upgradeTxns := make([]hexutil.Bytes, 0, 4) // 2 + 2 updates
+
+	// 1. Deploy CrossL2Inbox
+	deployCrossL2InboxTx, err := types.NewTx(&types.DepositTx{
+		SourceHash:          deployCrossL2InboxSource.SourceHash(),
+		From:                crossL2InboxDeployerAddress,
+		To:                  nil, // Contract creation
+		Mint:                big.NewInt(0),
+		Value:               big.NewInt(0),
+		Gas:                 420_000,
+		IsSystemTransaction: false,
+		Data:                crossL2InboxDeploymentBytecode,
+	}).MarshalBinary()
+	if err != nil {
+		return nil, err
+	}
+	upgradeTxns = append(upgradeTxns, deployCrossL2InboxTx)
+
+	// 2. Update CrossL2Inbox Proxy
+	updateCrossL2InboxProxyTx, err := types.NewTx(&types.DepositTx{
+		SourceHash:          updateCrossL2InboxProxySource.SourceHash(),
+		From:                common.Address{}, // From OVM_Deployer
```

Comment:
Where did you get the term `OVM_Deployer`? Just wondering because I do not recognize it

Reply:
That is AI slop, removing.